notebooks/AsymmetricErrors.ipynb

```diff
@@ -0,0 +1,577 @@
+      "text/plain": [
+       "<Figure size 640x480 with 1 Axes>"
+      ]
+     },
+     "metadata": {},
+     "output_type": "display_data"
+    }
+   ],
+   "source": [
+    "ui.reg_proj(pl.gamma, pl.ampl, nloop=[101, 101])\n",
+    "add_conf_gamma()"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "id": "2a9b27dd-afb9-44c0-83cf-8564d9ac52ba",
+   "metadata": {},
+   "source": [
+    "So, **all** the above used the average error values when calculating statistic values.\n",
+    "\n",
+    "The trick to using the asymmetric errors is to use the `resample_data` call to simulate drawing the data using the asymmetric errors and fitting the resulting simulated data. In the following we explicitly ask for 1000 iterations (although that is the default value) and fix the random-number seed. The call displays information on the distribution of the fit parameters:"
```

Comment:
In order to get the uncertainties on the fitted parameters we can use 'resample_data'........'simulated data. The bootstrap simulations assume one-sided Gaussian distributions for sampling each data point with sigma equal to elo, ehi.

Reply:
Also, you can link to  resample_data help which has the description:

> 
> Resample data with asymmetric error bars.
> 
> The function performs a parametric bootstrap assuming a skewed normal distribution centered on the observed data point with the variance given by the low and high measurement errors. The function simulates niter realizations of the data and fits each realization with the assumed model to obtain the best fit parameters. The function returns the best fit parameters for each realization, and displays the average and standard deviation for each parameter.

Reply:
I've reworked this paragraph to better describe the function. Hopefully it is a bit more useful now.

Reply:
Yes, it is better.